rhino-tools/src/main/resources/org/mozilla/javascript/tools/resources/Messages.properties

```diff
@@ -34,7 +34,7 @@ msg.shell.usage =\
     \    -w                 Enable warnings.\n\
     \    -version 100|110|120|130|140|150|160|170|180|200\n\
     \                       Set a specific language version.\n\
-    \    -opt [-1|0-9]      Set optimization level.\n\
+    \    -int               Run in interpreted mode.\n\
```

Comment:
'int' is confusing

Reply:
Yes, although don't you think that '-interpreted' is too long? And '-I' is too much like the "-I" argument in a C compiler.

What if we support both "-int" and "-interpreted" command-line flags?

Reply:
> although don't you think that '-interpreted' is too long?

sometimes, but i learned the hard way that is easier for me to type this many chars while writing the code & doc the spending the the answering all the questions ;-D